index.yaml

```diff
@@ -9775,6 +9775,76 @@ entries:
     - assets/rancher-grafana/rancher-grafana-6.6.401.tgz
     version: 6.6.401
   rancher-istio:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Istio
+      catalog.cattle.io/kube-version: '>= 1.25.0-0 < 1.29.0-0'
+      catalog.cattle.io/namespace: istio-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/permits-os: linux,windows
+      catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
+      catalog.cattle.io/release-name: rancher-istio
+      catalog.cattle.io/requests-cpu: 710m
+      catalog.cattle.io/requests-memory: 2314Mi
+      catalog.cattle.io/type: cluster-tool
+      catalog.cattle.io/ui-component: istio
+      catalog.cattle.io/upstream-version: 1.19.6
+    apiVersion: v1
+    appVersion: 1.19.6
+    created: "2024-02-22T19:54:03.62141129-03:00"
+    dependencies:
+    - condition: kiali.enabled
+      name: kiali
+      repository: file://./charts/kiali
+    - condition: tracing.enabled
+      name: tracing
+      repository: file://./charts/tracing
+    description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
+      for details.
+    digest: 5a15060443beea4b50d707539c68be830492a3474ccebd61a841085dab4048ff
```

Comment:
The digest here is different from the one in `dev-v2.7`

Reply:
Yes, as we previously discussed.
At the final step make validate was failing although the files were the same.
I had to execute make charts once again so it would stop failing.

I will deal with this in the final sync.